client/components/course/Sidebar/Relationship.vue

```diff
@@ -63,19 +69,27 @@ export default {
     associations() {
       const ids = this.getAssociationIds(this.activity);
       return filter(this.options, it => includes(ids, it.id));
+    },
+    displayType() {
```

Comment:
Always display type, but let's change multiselect options so they are either grouped by type or have a type as a smaller info/subtitle type of label.